python/GangaDirac/test/Unit/Utilities/TestDiracUtilitiesExternals.py

```diff
@@ -0,0 +1,78 @@
+import os
```

Comment:
You're right, they don't do anything DIRAC-specific (I was thrown off by the `assert timeit.timeit(` below).

The reason they are here in `GangaDirac` is a historical one that `execute` was originally written by Alex (I think) as part of `GangaDirac` and so the tests were placed alongside it. At some later point, the DIRAC-agnostic parts were factored out and moved to `Ganga` core but for some reason the tests remained. here. I'd be in favour of simply moving `GangaDirac/test/Unit/Utilities/TestDiracUtilities.py` to `Ganga/test/Unit/Utility/TestExecute.py`.